backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,68 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
+  id: ID! @primaryKey(sortKeyFields: [updatedAt])
+  game: Game! @hasOne
+  startTime: AWSDateTime
+  phaseOneTime: Int
+  phaseTwoTime: Int
+  teams: [Team] @hasMany
+  currentQuestion: Int!
+  isPhaseOne: GameSessionState!
+  updatedAt: AWSTimeStamp!
+}
+
+enum GameSessionState {
+	CHOOSINGTRICKANSWER
+	FINISHED
+	INITIALINTRO
+	NOTSTARTED
+	REVIEWINGRESULT
+	VOTING
+}
+
+type Game @model(subscriptions: null, timestamps: null) {
+  id: ID!
+  title: String
+  description: String
+  imageUrl: String
+  isAdvanced: Boolean
+  gameCode: Int
+  grade: String
+  questions: [Question] @hasMany
+}
+
+type Question @model(subscriptions: null, timestamps: null) {
+  id: ID!
+  question: String!
+  answer: String
+  trickAnswers: AWSJSON
+  imageUrl: String
+  instructions: AWSJSON
+  grade: String
+}
+
+type Team @model {
+  id: ID!
+  name: String!
+  question: Question! @hasOne
+  trickiestAnswerIDs: [ID]
+  teamMembers: [TeamMember] @hasMany
+  score: Int!
```

Comment:
Lets add `@default(value: 0)` directive here

Reply:
So this one doesn't seem to be a directive, all solutions for default values suggest to update the resolvers directly incase if it is missing

Reply:
Weirdly this one complains about type not being String. Looks like a bug on Amplify CLI
<img width="309" alt="Screen Shot 2022-03-09 at 11 21 02 AM" src="https://user-images.githubusercontent.com/47463734/157484136-78babf38-463e-4ee4-8a1a-e597909a7372.png">

Reply:
Huh, bummer. Feel free to open an issue with Amplify team if you were interested